Kitodo-API/src/main/java/org/kitodo/api/dataformat/SortedList.java

```diff
@@ -30,75 +32,109 @@
     private Function<T, Integer> orderGetter;
 
     /**
-     * Constructor for a sorted list.
+     * Constructor for an order-aware list.
      * 
      * @param orderGetter
-     *            possibility to get the order number for an element
+     *            possibility to get an order number for T
      */
     public SortedList(Function<T, Integer> orderGetter) {
         this.orderGetter = orderGetter;
     }
 
+    /**
+     * Sorts the list by the media unit’s order value and can handle duplicate
+     * occurrences of the same number (unlike a TreeMap). The order of list
+     * items is changed only when necessary. The order of list members with the
+     * same ordinal number is not affected.
+     */
+    private void ensureOrder() {
```

Comment:
Didn't you remove this stuff? I thing a rebase is neccessary...

Reply:
Normally git works and does amazing things. But here it has caught the wrong file version on the rebase for some unknown reason. I have undone that.